### PR TITLE
Update `Card` to use environment value to drive the content layout

### PR DESCRIPTION
**Pull request by PavelHolec**
This combines previous flexibility with the ability to configure multiple Cards via environment.

### CODE REVIEW

File: Sources/Orbit/Components/Card.swift [hunk 1]
```diff
@@ -192,6 +186,39 @@ public extension AccessibilityID {
...
+// MARK: - Types
+
+/// The layout of Orbit ``Card`` stack content.
+public enum CardLayout {
+    
+    /// Content layout uses a `VStack` with customizable padding and spacing.
+    case custom(alignment: HorizontalAlignment = .leading, padding: CGFloat = .medium, spacing: CGFloat = .medium)
+    
+    /// Content layout uses a `VStack` with default `.medium` padding and spacing.
+    public static var `default` = Self.custom()
```
Comment (sjavora) [inline Sources/Orbit/Components/Card.swift:198]: `let`

File: Sources/Orbit/Components/Card.swift [hunk 2]
```diff
@@ -192,6 +186,39 @@ public extension AccessibilityID {
     static let cardAction               = Self(rawValue: "orbit.card.action")
 }
 
+// MARK: - Types
+
+/// The layout of Orbit ``Card`` stack content.
+public enum CardLayout {
+    
+    /// Content layout uses a `VStack` with customizable padding and spacing.
+    case custom(alignment: HorizontalAlignment = .leading, padding: CGFloat = .medium, spacing: CGFloat = .medium)
```
Comment (sjavora) [inline Sources/Orbit/Components/Card.swift:195]: Do you anticipate more `case`s here? If not, there isn't really a reason for this to be an `enum`.

**Review by sjavora — APPROVED** [review]

File: Sources/Orbit/Components/Card.swift (hunk 2 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/Card.swift:195]: I was thinking about future cases like `lazyStack` (which would use iOS14+ LazyVStack for both stacks that are currently used) etc. But it makes sense to keep it as a struct for now.

File: Sources/Orbit/Components/Card.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/Card.swift]: Updated

File: Sources/Orbit/Components/Card.swift (hunk 2 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/Card.swift]: Replaced with a struct with two public predefined values (`default` and `fill`)

**Review by sjavora — APPROVED** [review]

**Pull request merged**